tests/test_annotation_stores.py

```diff
@@ -25,19 +33,24 @@
     SQLiteStore,
 )
 from tiatoolbox.annotation.storage import SQLiteMetadata
+from tiatoolbox.enums import GeometryType
 
 if TYPE_CHECKING:  # pragma: no cover
     from numbers import Number
 
 sqlite3.enable_callback_tracebacks(True)
```

Comment:
Please can you specify the variable name for the boolean argument instead of positional argument?

Reply:
I cannot do that because this function does not take key word arguments.

Reply:
ok may be we can add noqa flag in #686 #677